### DISCUSSION TITLE
Ghost notes and invisible sliders in mania editor

### DISCUSSION THREAD

**Discussion by papertek** [ppy/osu#20646] [Q&A] [answered]
while messing around i switched from argon to classic and went into the editor, after scrolling a bit notes started to disappear and the left key didn't stop glowing

https://user-images.githubusercontent.com/72106503/194723694-159d0a35-3935-4e81-a97f-ca00ca6b507e.mp4

**Comment by smoogipoo** [comment]
Does this happen in gameplay at all?

**Reply by papertek** [reply]
tried playtesting it in the editor and no, it doesn't happen during gameplay

**Comment by peppy** [comment]
Potentially similar to https://github.com/ppy/osu/issues/20644

**Reply by papertek** [reply]
checked it out with the latest release (2022.1101.0) and i can still reproduce

**Comment by fledermaus101** [comment]
I can reliably replicate the behavior by going to a section with LNs. After that skipping to an unrelated far away location and returning to the LNs.

**Reply by papertek** [reply]
ok

**Comment by peppy** [comment, marked as answer]
Tracking via https://github.com/ppy/osu/issues/21090.
